core/src/main/java/org/apache/gravitino/listener/EventBus.java

```diff
@@ -64,7 +73,15 @@ public void dispatchEvent(Event event) {
    *     EventBus.
    */
   @VisibleForTesting
-  List<EventListenerPlugin> getPostEventListeners() {
-    return postEventListeners;
+  List<EventListenerPlugin> getEventListeners() {
+    return eventListeners;
+  }
+
+  private void dispatchPostEvent(Event postEvent) {
+    eventListeners.forEach(postEventListener -> postEventListener.onPostEvent(postEvent));
+  }
+
+  private void dispatchPreEvent(PreEvent preEvent) throws ForbiddenException {
+    eventListeners.forEach(preEventListener -> preEventListener.onPreEvent(preEvent));
```

Comment:
Actually, we don't differentiate pre or post event listeners, the code here is a little confusing. We can change to:

```java
  private void dispatchPostEvent(Event postEvent) {
    eventListeners.forEach(listener -> listener.onPostEvent(postEvent));
  }
```

Reply:
ok

Reply:
done